system/ee/installer/updates/ud_4_00_05.php

```diff
@@ -44,6 +44,7 @@ public function do_update()
     private function fixGridModifierParsing()
     {
         $grid_fields = ee('Model')->get('ChannelField')
+            ->fields('field_id')
```

Comment:
@intoeetive Just want to make sure this is here on purpose?  (since it's the update for EE version 4.0.5) ?

Reply:
Yes it's on purpose (it limits the fields that are being fetched to only those that are needed by this update script)